src/trace_context.js

```diff
@@ -10,18 +10,31 @@ let namespace;
 
 /**
  * Creates an active context for tracer and run the handle
- * @param {Object} tracer object
+ * @param {Function} createTracer create a tracer object
  * @param {Function} handle function to run the context in
- * @param {Function} The runner event
  */
-function RunInContext(tracer, handle) {
+function RunInContext(createTracer, handle) {
```

Comment:
Does it matter if this function will always return the value from `handle` function? It seems redundant to have two functions

Reply:
You are right :)